extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -38,7 +38,7 @@ export class ResourceTypePickerDialog extends DialogBase {
 		this._selectedResourceType = resourceType;
 		this._installToolButton = azdata.window.createButton(localize('deploymentDialog.InstallToolsButton', "Install tools"));
 		this._toDispose.push(this._installToolButton.onClick(() => {
-			this.installTools();
+			this.installTools().catch(() => {/* Empty catch to keep linter happy */ });
```

Comment:
Is there a reason you wouldn't at least console.log this? In the unlikely event that an error occurs it'd be good to have something to go on